schema/google/showcase/v1beta1/echo.proto

```diff
@@ -100,6 +100,16 @@ service Echo {
     };
   }
 
+  // This method always fails with a gRPC error status that contains a series of
+  // error details: the standard ones in error_details.proto plus a custom,
+  // Showcase-defined PoetryError.
+  rpc FailEchoWithDetails(FailEchoWithDetailsRequest) returns (FailEchoWithDetailsResponse) {
+    option (google.api.http) = {
+      post: "/v1beta1/echo:fail-with-details"
```

Comment:
```suggestion
      post: "/v1beta1/echo:failWithDetails"
```
I know `EchoErrorDetails` has a hyphen in the custom method suffix, but it shouldn't either. We use lowerCamelCase for custom method suffixes 
![image](https://github.com/user-attachments/assets/a88a5fa5-963d-4a99-b9f2-071f13c9eaf4)

Reply:
Let's not bother, id rather not break anyone, maybe just add a comment saying this is non standard?